tests/end2end/features/test_prompts_bdd.py

```diff
@@ -37,6 +39,20 @@ def wait_for_prompt(quteproc):
     quteproc.wait_for(message='Asking question *')
 
 
+@bdd.given("I may need a fresh instance")
+def fresh_instance(quteproc):
+    """Restart qutebrowser to bypass webengine's permission persistance."""
+    # Qt6.8 by default will remember feature grants or denies. When we are
+    # on PyQt6.8 we disable that with the new API, otherwise restart the
+    # browser to make it forget previous prompts.
+    if (
+        qtutils.version_check("6.8", compiled=False)
```

Comment:
```suggestion
        qtutils.version_check("6.8", compiled=True)
```

I think? It's not enough that we have a Qt 6.8, but PyQt needed to be compiled against Qt 6.8 to have the API (but also see below).

Reply:
Hmm, I think it's right as-is. I'm looking for the case where we have runtime 6.8 but can't call the new APIs to disable persistence, so we restart instead and the file gets cleared.